mteb/leaderboard/table.py

```diff
@@ -101,7 +101,7 @@ def get_means_per_types(df: pd.DataFrame) -> pd.DataFrame:
 def failsafe_get_model_meta(model_name):
     try:
         return get_model_meta(model_name)
-    except Exception as e:
+    except Exception:
```

Comment:
Since your PR is not concerned with the leaderboard, you probably shouldn't put changes in it related to that.

Reply:
Yes, I believe that was a result of running make lint, however I can leave that out.